roles/amq_broker/tasks/upgrade.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: "Update profile for instance {{ amq_broker.instance_name }} with new artemis version"
+  ansible.builtin.lineinfile:
+    path: "{{ amq_broker.instance_home }}/etc/artemis.profile"
+    regexp: '^ARTEMIS_HOME=.*'
+    line: "ARTEMIS_HOME='{{ amq_broker.home }}'"
+  become: yes
+  register: broker_updated
+  notify:
+    - restart amq_broker
+
+- name: "Check artemis {{ amq_broker.home }} activemq schema"
+  xml:
```

Comment:
@guidograzioli missing the ansible.builtin. prefix (or the one from the collection)?

Reply:
it's indeed not builtin, but from community.general